content/pages/stories.html

```diff
@@ -0,0 +1,235 @@
+---
+$title: Stories
+
+stylesheet: pages/stories.css
+components:
+  - carousel
+  - video
+---
+
+{% set stories = g.doc('/content/includes/stories.yaml', locale=doc.locale) %}
+
+<div class="stories">
+  <div class="hero-accent"></div>
+  <div class="hero-background">
+    <amp-img
+      noloading
+      height="{{stories.hero.background_image_height}}"
+      width="{{stories.hero.background_image_width}}"
+      src="{{stories.hero.background_image}}"
+      layout="responsive">
+    </amp-img>
+  </div>
+  <div class="container xl">
+    <section class="hero">
+      <div class="text">
+        <div class="title-wrap">
+          <div class="icon">
+            <amp-img
+              noloading
+              height={{stories.hero.icon_height}}
+              width={{stories.hero.icon_width}}
+              src="{{stories.hero.icon}}"
+              layout="responsive">
+            </amp-img>
+          </div>
+          <h1>{{_(stories.hero.title)}}</h1>
+        </div>
+        <div class="mobile-image">
+          <amp-img
```

Comment:
Should be using new hero image in mobile view too.  We tested mobile view and it's the old hero image.

Reply:
@bpaduch happy to do that but figured that since the mobile hero won't be animated it might be nicer to have to old version, which, while static, tells more of the story. LMK.

Reply:
Team wants the same animated hero seen on desktop for mobile.

Reply:
Cool! The only issue is that that's a video so a lot of phones will not autoplay/play inline. LMK if that's okay @bpaduch

Reply:
Odd, I thought they would, like in the Showcases page.  Make sure to specify a fallback poster for the amp-video.

Reply:
Sounds good!